.github/scripts/js/ci.js

```diff
@@ -921,92 +1093,160 @@ const startBuildAndTestWorkflow = async ({ github, context, core }) => {
     return core.setFailed(`Create issue comment failed: ${JSON.stringify(response)}`);
   }
 
-  // Start 'release-build-and-test' workflow.
-  console.log('Start workflow.');
-  const issue_id = '' + issue.id;
-  const issue_number = '' + issue.number;
-  const comment_id = '' + response.data.id;
-  response = await github.rest.actions.createWorkflowDispatch({
-    owner: context.repo.owner,
-    repo: context.repo.repo,
-    workflow_id: 'build-and-test_release.yml',
-    ref: github_ref,
-    inputs: { issue_id, issue_number, comment_id }
-  });
-  if (response.status < 200 || response.status >= 300) {
-    return core.setFailed(`Error calling dispatch. Response: ${JSON.stringify(response)}`);
+  return {
+    issue_id: '' + issue.id,
+    issue_number: '' + issue.number,
+    comment_id: '' + response.data.id,
   }
-};
+}
 
 /**
- * Start build-and-test_release workflow.
+ * Start workflow using workflow_dispatch event.
  *
- * @param {object} inputs
- * @param {object} inputs.github - A pre-authenticated octokit/rest.js client with pagination plugins.
- * @param {object} inputs.context - An object containing the context of the workflow run.
- * @param {object} inputs.core - A reference to the '@actions/core' package.
+ * @param {object} args
+ * @param {object} args.github - A pre-authenticated octokit/rest.js client with pagination plugins.
+ * @param {object} args.context - An object containing the context of the workflow run.
+ * @param {object} args.core - A reference to the '@actions/core' package.
+ * @param {object} args.workflow_id - A name of the workflow YAML file.
+ * @param {object} args.ref - A Git ref.
+ * @param {object} args.inputs - Inputs for the workflow_dispatch event.
  * @returns {Promise<void>}
  */
-const startBuildAndTestWorkflowNoComment = async ({ github, context, core }) => {
-  const github_ref = context.ref;
+const startWorkflow = async ({ github, context, core, workflow_id, ref, inputs }) => {
+  core.info(`Start workflow '${workflow_id}' using ref '${ref}' and inputs ${JSON.stringify(inputs)}.`);
 
-  // Start 'release-build-and-test' workflow.
-  console.log('Start workflow.');
-  response = await github.rest.actions.createWorkflowDispatch({
```

Comment:
Please, don't refer to globals. Should be `let/const response` here.

Reply:
Missed during the code splitting.